app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionDetailFragment.java

```diff
@@ -85,6 +79,17 @@ private void initTheme() {
                     new ActivityManager.TaskDescription(viewModel.getSessionTitle(), null,
                             ContextCompat.getColor(activity, viewModel.getSessionVividColorResId()));
             activity.setTaskDescription(taskDescription);
+
+            // Change status bar scrim color
+            Window window = activity.getWindow();
```

Comment:
I think it's not necessary 🤔 
`activity.setTheme(viewModel.getTopicThemeResId());` is enough to change the theme.
Actually, I could change the status color on DroidKaigi2016 app.
https://github.com/konifar/droidkaigi2016/blob/master/app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java#L61